spec/models/vacancy_spec.rb

```diff
@@ -433,4 +433,18 @@
       expect(vacancy.education).to eq(sanitized_education)
     end
   end
+
+  describe '#to_row' do
+    let(:vacancy) { build(:vacancy) }
```

Comment:
You should be able to use `create(:vacancy)` instead of `build` and `save!` in the before block as the default factory has all required values set.